pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -860,6 +863,19 @@ type Volume struct {
 	Encrypted *bool `json:"encrypted,omitempty"`
 }
 
+// CRI contains information about the Container Runtimes.
+type CRI struct {
+	// The name of the CRI library
+	Name CRIName `json:"name"`
```

Comment:
Can we also already add an optional `providerConfig` field like for other extension types:

https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1beta1/types_shoot.go#L251-L253